### DISCUSSION TITLE
Send Network Request Actions & HTTPS

### DISCUSSION THREAD

**Discussion by XtaTonGo** [RavenSystem/esp-homekit-devices#2308] [Q&A] [answered]
Hello,
some time ago i migrated my webserver (node-red) from HTTP to HTTPS and the notification actions stopped working..

If anybody knows does "Send Network Request Actions" supports TLS? For now I need to send some information to the host with **HTTPS** requests instead of HTTP.

Here is a part of my config for Garage Door:
`"0":{"r":[[4,1,0.5]],"h":[{"m":0,"h":"iot.home.lan","p":1880,"u":"came?cmd=0","w":"1"}]},

**Comment by RavenSystem** [comment, marked as answer]
HTTPS is currently not supported. It will be supported in the future, but not for ESP8266.

**Reply by XtaTonGo** [reply]
Thank you for information
